spring-kafka/src/main/java/org/springframework/kafka/core/DefaultKafkaConsumerFactory.java

```diff
@@ -30,32 +31,43 @@
 import org.springframework.util.StringUtils;
 
 /**
- * The {@link ConsumerFactory} implementation to produce a new {@link Consumer} instance
- * for provided {@link Map} {@code configs} and optional {@link Deserializer} {@code keyDeserializer},
- * {@code valueDeserializer} implementations on each {@link #createConsumer()}
+ * The {@link ConsumerFactory} implementation to produce new {@link Consumer} instances
+ * for provided {@link Map} {@code configs} and optional {@link Deserializer}s on each {@link #createConsumer()}
  * invocation.
+ * <p>
+ * If you are using {@link Deserializer}s that have no-arg constructors and require no setup, then simplest to
+ * specify {@link Deserializer} classes in spring.kafka.consumer configuration and create the
```

Comment:
First of all the `spring.kafka.consumer` should be wrapped to `{@code}` to make it more visible.
However on the other hand it is wrong to say about this property since it is part of Spring Boot. 
I think we need to be more specific here and really say about a native Apache Kafka property we are aiming to - `ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG` and/or `ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG`.

Reply:
Yes, that makes sense. I'll change it.